### DISCUSSION TITLE
Set editor camera start position

### DISCUSSION THREAD

**Discussion by Oreztov** [diesel-modding/BeardLib-Editor#646] [Implemented Ideas]
What I'd like is the ability to set the editor camera's start position (i.e. when you load into the map, you can choose where the camera starts at).
I'm currently working on a map with a playable area that's a lot higher than 0,0,0 and this feature would be very helpful, instead of having to fly up every single time I load into the map.

**Comment by RexTheWho** [comment]
I like this idea, even just having the camera return to its last position when you quit would be handy for reloading to update scripting changes or undo unit sequences done in editor.
